src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -27,33 +30,43 @@ import Image from '../Image';
 import ReportActionItemImage from './ReportActionItemImage';
 import * as TransactionUtils from '../../libs/TransactionUtils';
 import OfflineWithFeedback from '../OfflineWithFeedback';
+import categoryPropTypes from '../categoryPropTypes';
 
 const propTypes = {
+    /** Whether we should display the horizontal rule below the component */
+    shouldShowHorizontalRule: PropTypes.bool.isRequired,
+
+    /* Onyx Props */
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
     /** The report currently being looked at */
     report: reportPropTypes.isRequired,
```

Comment:
nitpick, but report is not an Onyx prop in this case, it's passed as a prop and not from withOnyx

Reply:
Done - 37d6770.